tests/Makefile.am.inc

```diff
@@ -109,6 +109,9 @@ dist_installed_test_dbs_DATA = tests/dbs/no_tables
 dist_installed_test_portals_DATA = tests/portals/test.portal
 endif
 
+tests/test-document-fuse.sh: tests/test-document-fuse.sh.in
+	$(AM_V_GEN) $(SED) -e "s|\@libexecdir\@|$(libexecdir)|" $< > $@
```

Comment:
It seems confusing that this is replacing `@libexecdir@` as an Autoconf-style substitution, while leaving `@testdir@` as a literal string.

Reply:
Indeed. Should I add a comment or escape it as `@@testdir@@`?

Reply:
I'd prefer to see whether we can make it unnecessary first.